datafusion/core/tests/sqllogictests/src/engines/datafusion/normalize.rs

```diff
@@ -36,7 +35,7 @@ pub fn convert_batches(batches: Vec<RecordBatch>) -> Result<Vec<Vec<String>>> {
         let mut rows = vec![];
         for batch in batches {
             // Verify schema
-            if !equivalent_names_and_types(&schema, batch.schema()) {
+            if !schema.contains(&batch.schema()) {
```

Comment:
It appears `contains` is stricter than the `equivalent_names_and_types` (it also verifies nullability, metadata and dictionary ids). I think this is fine but wanted to point it out.

https://docs.rs/arrow/36.0.0/arrow/datatypes/struct.Schema.html#method.contains


https://docs.rs/arrow-schema/36.0.0/src/arrow_schema/field.rs.html#434

Reply:
Thanks for checking. It is stricter thats correct but in the same time it has more weak rules for nullability checks which work for us now. 

```
        // self need to be nullable or both of them are not nullable
        && (self.nullable || !other.nullable)
```